<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd">
<suite parallel="tests" name="Failed suite [SubmitOrderSuite]" guice-stage="DEVELOPMENT" verbose="0">
  <test thread-count="5" parallel="tests" name="ErrorValidationTest(failed)" verbose="0">
    <groups>
      <run>
        <include name="ErrorHandling"/>
      </run>
    </groups>
    <classes>
      <class name="com.test.Error_Validation_Test">
        <methods>
          <include name="login_error_Validation"/>
          <include name="teardown"/>
          <include name="launchApplication"/>
        </methods>
      </class> <!-- com.test.Error_Validation_Test -->
    </classes>
  </test> <!-- ErrorValidationTest(failed) -->
</suite> <!-- Failed suite [SubmitOrderSuite] -->
